proto/xudrpc.proto

```diff
@@ -416,6 +439,23 @@ message Balance {
   uint64 unconfirmed_wallet_balance = 6 [json_name = "unconfirmed_wallet_balance"];
 }
 
+message BalanceAlert {
+  // The total balance.
+  uint64 total_balance = 1 [json_name = "total_balance"];
+  // The side of the low balance.
+  enum Side {
+    REMOTE = 0;
+    LOCAL = 1;
+  }
+  Side side = 2 [json_name = "side"];
+  // The bound of the low balance in percentage.
+  uint32 bound = 3 [json_name = "bound"];
```

Comment:
Maybe this should be `percent` instead of `bound`? Seems a bit more clear to me. And the comment can say `The percent of the total channel capacity that is on this side of the channel`

Reply:
I changed comment as `The percent of the total trading balance.` since this's actually trading balance alert, but changed the field name as `percent`